src/main/java/org/broadinstitute/hellbender/tools/recalibration/RecalDatum.java

```diff
@@ -15,6 +15,7 @@
     public static final byte MAX_RECALIBRATED_Q_SCORE = SAMUtils.MAX_PHRED_SCORE;
     private static final double UNINITIALIZED = -1.0;
     private static final long serialVersionUID = 1L;
+    private static final double MULTIPLIER = 1000.0;
```

Comment:
The changes here to `RecalDatum` are in https://github.com/broadinstitute/hellbender/pull/878, right? Are you going to merge that version first, and rebase this one away?